### DISCUSSION TITLE
result of flutter format are different between flutter-action and local environment

### DISCUSSION THREAD

**Discussion by suesitran** [subosito/flutter-action#217] [General]
Link to action: https://github.com/suesitran/interactive_diary/runs/8240806582?check_suite_focus=true

Current flutter-action version: 2.6.2
flutter version use: 3.3.1

<details>

<summary>Output from local machine</summary>

suesi$ flutter format . --set-exit-if-changed
Formatted 71 files (0 changed) in 3.88 seconds.

</details>

<details>
<summary>Flutter doctor - local machine</summary>

[✓] Flutter (Channel stable, 3.3.1, on macOS 11.6.8 20G730 darwin-x64, locale en-AU)
    • Flutter version 3.3.1 on channel stable at /Users/suesi/Development/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 4f9d92fbbd (26 hours ago), 2022-09-06 17:54:53 -0700
    • Engine revision 3efdf03e73
    • Dart version 2.18.0
    • DevTools version 2.15.0

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/suesi/Library/Android/sdk
    • Platform android-31, build-tools 30.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.2.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 13C100
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2021.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)

[✓] VS Code (version 1.53.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.20.0

[✓] Connected device (3 available)
    • iPhone 13 (mobile) • ABFF7038-DB26-4C5A-81B0-947CC7EBEAC2 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-15-2 (simulator)
    • macOS (desktop)    • macos                                • darwin-x64     • macOS 11.6.8 20G730 darwin-x64
    • Chrome (web)       • chrome                               • web-javascript • Google Chrome 105.0.5195.102

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!

</details>

<summary>Output from flutter-action</summary>

```
##[group***Run flutter format . --set-exit-if-changed
flutter format . --set-exit-if-changed
shell: /bin/bash -e ***0***
env:
  JAVA_HOME: /Users/runner/hostedtoolcache/Java_Zulu_jdk/11.0.16-8/x64
  FLUTTER_ROOT: /Users/runner/hostedtoolcache/flutter/stable-3.3.1-x64
  PUB_CACHE: /Users/runner/hostedtoolcache/flutter/stable-3.3.1-x64/.pub-cache
##[endgroup***

  ╔════════════════════════════════════════════════════════════════════════════╗
  ║                 Welcome to Flutter! - https://flutter.dev                  ║
  ║                                                                            ║
  ║ The Flutter tool uses Google Analytics to anonymously report feature usage ║
  ║ statistics and basic crash reports. This data is used to help improve      ║
  ║ Flutter tools over time.                                                   ║
  ║                                                                            ║
  ║ Flutter tool analytics are not sent on the very first run. To disable      ║
  ║ reporting, type 'flutter config --no-analytics'. To display the current    ║
  ║ setting, type 'flutter config'. If you opt out of analytics, an opt-out    ║
  ║ event will be sent, and then no further information will be sent by the    ║
  ║ Flutter tool.                                                              ║
  ║                                                                            ║
  ║ By downloading the Flutter SDK, you agree to the Google Terms of Service.  ║
  ║ Note: The Google Privacy Policy describes how data is handled in this      ║
  ║ service.                                                                   ║
  ║                                                                            ║
  ║ Moreover, Flutter includes the Dart SDK, which may send usage metrics and  ║
  ║ crash reports to Google.                                                   ║
  ║                                                                            ║
  ║ Read about data we send with crash reports:                                ║
  ║ https://flutter.dev/docs/reference/crash-reporting                         ║
  ║                                                                            ║
  ║ See Google's privacy policy:                                               ║
  ║ https://policies.google.com/privacy                                        ║
  ╚════════════════════════════════════════════════════════════════════════════╝

Running "flutter pub get" in flutter_tools...                    [2](https://github.com/suesitran/interactive_diary/runs/8240806582?check_suite_focus=true#step:5:2),5[3](https://github.com/suesitran/interactive_diary/runs/8240806582?check_suite_focus=true#step:5:3)3ms
Formatted packages/nartus_ui_package/lib/theme/nartus_color.dart
Formatted packages/nartus_ui_package/lib/theme/nartus_theme.dart
Formatted [5](https://github.com/suesitran/interactive_diary/runs/8240806582?check_suite_focus=true#step:5:5)[6](https://github.com/suesitran/interactive_diary/runs/8240806582?check_suite_focus=true#step:5:6) files (2 changed) in 1.[10](https://github.com/suesitran/interactive_diary/runs/8240806582?check_suite_focus=true#step:5:10) seconds.
Formatting failed: 1
##[error***Process completed with exit code 1.

```

</details>

**Comment by subosito** [comment]
@suesitran Please add the below line, and post the result here:

```yaml
- run: flutter format -o show packages/nartus_ui_package/lib/theme/
```

**Comment by suesitran** [comment]
@subosito here is the link that runs with above comment: https://github.com/suesitran/interactive_diary/runs/8261237425?check_suite_focus=true

**Comment by suesitran** [comment]
i @subosito , any update?
